installers/MacInstaller.hs

```diff
@@ -34,6 +34,8 @@ main = do
   copyFile "cardano-launcher" (dir <> "/cardano-launcher")
   copyFile "cardano-node" (dir <> "/cardano-node")
   copyFile "wallet-topology.yaml" (dir <> "/wallet-topology.yaml")
+  copyFile "configuration.yaml" (dir <> "/configuration.yaml")
```

Comment:
This file should be copied from cardano-sl bridge, otherwise we'll have to keep it in sync in daedalus was well, that's 3 places including iohk-ops.

Reply:
@domenkozar, acknowledged